packages/wallet-service/tests/db.test.ts

```diff
@@ -479,10 +479,10 @@ test('initWalletBalance', async () => {
     { address: addr3, txId: tx3, tokenId: token2, balance: 11, timestamp: ts3 },
   ];
   const addressEntries = [
-    // address, tokenId, unlocked, locked, lockExpires, transactions
-    [addr1, token1, 2, 0, null, 2, 0, 0, 4],
-    [addr1, token2, 1, 4, timelock, 1, 0, 0, 5],
-    [addr2, token1, 5, 2, null, 2, 0, 0, 20],
+    // address, tokenId, unlocked, locked, lockExpires, unlocked_authorities, locked_authorities, transactions
+    [addr1, token1, 2, 0, null, 2, 1, 0, 4],
```

Comment:
question(non-blocking): The amount of columns on the comment is 8. However, there are 9 columns on the addressEntries code validation.

Should we add a comment explaining this extra column, or an explanation for the difference?

Reply:
Fixed the comment in 1fa63fb560454e5affeda35ca593d0bba8da09fe